src/gt4py/testing/suites.py

```diff
@@ -22,6 +22,7 @@
 import gt4py as gt
 from gt4py import gtscript
 from gt4py import storage as gt_storage
+import gt4py.definitions as gt_defs
```

Comment:
Should this become a standard throughout? Elsewhere I've seen
```
import gt4py.definitions as gt_definitions
```

Reply:
We didn't pay too much attention to the local alias of packages in the past, but I agree it helps to keep a consistent naming so I fixed it here.